tools/nxfuse/Makefile

```diff
@@ -0,0 +1,164 @@
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+# ==========================================================================
+#   Copyright (C) 2016 Ken Pettit. All rights reserved.
+#   Author: Ken Pettit <pettitkd@gmail.com>
+#
+#   nxfuse is a Linux FUSE filesystem that allows native mounting of
+#          NuttX filesystems within Linux.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in
+#    the documentation and/or other materials provided with the
+#    distribution.
+# 3. Neither the name NuttX nor the names of its contributors may be
+#    used to endorse or promote products derived from this software
+#    without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+# FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+# COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+# OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+# AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+# ==========================================================================
+
+.silent:
+
+-include .config
+
+TINYARADIR    ?= ../../os
+
+APPNAME     = nxfuse
+
+OBJDIR		=  obj
+DEPDIR		=  dep
+SRCDIR      =  src
+
+CC          =  $(CROSS_COMPILE)gcc
+LDFLAGS     +=  -g
+LIBFILES    +=  -lm -lpthread -lfuse
+ifeq ($(RELEASE),)
+CFLAGS      +=  -g -Wall -I include -DFAR= -DTRUE=1 -DFALSE=0 -Wno-unused-value -D_FILE_OFFSET_BITS=64
+else
+CFLAGS      +=  -O2 -Wall -I include -DFAR= -DTRUE=1 -DFALSE=0 -Wno-unused-value -D_FILE_OFFSET_BITS=64
+endif
+
+SOURCES     =  $(wildcard $(SRCDIR)/*.c)
+
+ifeq ($(CONFIG_FS_SMARTFS),y)
+SOURCES     += $(wildcard $(SRCDIR)/smartfs/*.c)
+endif
+
+SRCTMP      =  $(patsubst $(SRCDIR)/%,%,$(SOURCES))
+OBJTMP		=  $(SRCTMP:.c=.o)
```

Comment:
please correct indentation in whole file

Reply:
Done.

Reply:
Done